deployment/deployContracts.js

```diff
@@ -370,3 +411,93 @@ main().catch((e) => {
     console.error(e);
     process.exit(1);
 });
+
+async function deployPolygonZkEVMDeployer(deployer) {
+    const PolgonZKEVMDeployerFactory = await ethers.getContractFactory('PolygonZkEVMDeployer', deployer);
+
+    const deployTxZKEVMDeployer = (PolgonZKEVMDeployerFactory.getDeployTransaction(
+        deployer.address,
+    )).data;
+
+    const gasLimit = ethers.BigNumber.from(1000000); // Put 1 Million, aprox 650k are necessary
```

Comment:
I would add this `gasLimit` & `gasPrice` as a constants at the beginning of the script

Reply:
I think it's ok this way, i will have to do a different script for mainnet anyway, there i will put in in other format ^^